src/components/Board/Board.container.js

```diff
@@ -718,6 +718,32 @@ export class BoardContainer extends Component {
     }
   };
 
+  handleLayoutChange = (currentLayout, layouts) => {
+    const { updateBoard, replaceBoard, board } = this.props;
+    currentLayout.sort((a, b) => {
```

Comment:
I believe you can do the same thing with `_.sortBy(layouts, ['y', 'x']);`. It won't modify the array in place though.